vcsclient/bitbucketcloud.go

```diff
@@ -224,6 +224,54 @@ func (client *BitbucketCloudClient) GetLatestCommit(ctx context.Context, owner,
 	return CommitInfo{}, nil
 }
 
+func (client *BitbucketCloudClient) GetCloneInfo(ctx context.Context, owner, repository string) (CloneInfo, error) {
+	if err := validateParametersNotBlank(owner, repository); err != nil {
```

Comment:
This function changed recently

Reply:
Thanks, I have rebased on recent changes and align with new approach.